domain/ide/workspace/folder.go

```diff
@@ -194,7 +194,7 @@ func (f *Folder) filterCachedDiagnostics() (fileIssues map[string][]snyk.Issue)
 
 func filterIssues(issues []snyk.Issue, supportedIssueTypes map[product.FilterableIssueType]bool) []snyk.Issue {
 	logger := log.With().Str("method", "filterIssues").Logger()
-	filteredIssues := []snyk.Issue{}
+	filteredIssues := make([]snyk.Issue, 0)
```

Comment:
Why?

Reply:
Goland likes it better that way - and I don't have any clue why.